src/cromshell/utilities/CromshellConfig.py

```diff
@@ -0,0 +1,127 @@
+import logging
```

Comment:
I think a more pythonic way of coding this module would be to remove the class itself and define everything at the module level.  We can talk about it at the meeting.

Reply:
done